mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/mapping/impl/BridgeResolver.java

```diff
@@ -325,6 +327,14 @@ private void addDefaults() {
 					.valueBinder( new StaticValueBinder<>( GeoPoint.class, DefaultGeoPointBridge.INSTANCE ) );
 			exactType( GeoPoint.class )
 					.identifierBridge( DefaultGeoPointBridge.INSTANCE );
+
+			// arrays for vector fields:
+			exactType( byte[].class )
+					.valueBridge( DefaultByteArrayBridge.INSTANCE )
+					.identifierBridge( DefaultByteArrayBridge.INSTANCE );
+			exactType( float[].class )
+					.valueBridge( DefaultFloatArrayBridge.INSTANCE )
+					.identifierBridge( DefaultFloatArrayBridge.INSTANCE );
```

Comment:
I don't really see the point of supporting identifiers of type `byte[]`/`float[]`, but... ok.

This makes me think, though: you'll probably need to add entries to `PropertyTypeDescriptor#all` to get these bridges tested.